ci/prod/build/frontend.Dockerfile

```
@@ -0,0 +1,22 @@
+
+COPY . /app
+
+ARG FRONTEND_PUBLIC_API_URL
+ENV FRONTEND_PUBLIC_API_URL=$FRONTEND_PUBLIC_API_URL
+
+ARG FRONTEND_PUBLIC_SENTRY_RELEASE
+ENV FRONTEND_PUBLIC_SENTRY_RELEASE=$FRONTEND_PUBLIC_SENTRY_RELEASE
+
+ENV NODE_ENV=production
+
+RUN --mount=type=secret,id=build_secret export $(cat /run/secrets/build_secret | xargs) && \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)